_playwright-tests/UI/Templates.spec.ts

```diff
@@ -8,10 +8,27 @@ test.describe('Templates', () => {
   }) => {
     await navigateToTemplates(page);
     await closePopupsIfExist(page);
-
-    const AddButton = page.locator('[data-ouia-component-id="create_content_template"]');
-
-    // Wait for the Add button to become enabled (up to 10 seconds)
-    await AddButton.first().isEnabled({ timeout: 10000 });
+    await page.getByRole('button', { name: 'Add content template' }).click();
+    const architectureFilterButton = page.getByRole('button', { name: 'Select architecture' });
+    await architectureFilterButton.click()
+    await page.getByRole('option', { name: 'x86_64' }).click();
+    const versionFilterButton = page.getByRole('button', { name: 'Select version' });
+    await versionFilterButton.click();
+    await page.getByRole('menuitem', { name: 'el8' }).locator('label').click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    // select Red hat repos Checkbox optional
+    await page.getByRole('button', { name: 'Next' }).click();
+    // select checkbox for cutom repo optional
+    page.getByRole('button', { name: 'Next' }).click();
+    await page.getByRole('radio', { name: 'use-latest-snapshot' }).fill('True');
```

Comment:
`.fill(True)` ? I though we should be using https://playwright.dev/docs/input#checkboxes-and-radio-buttons or is that not working?

Reply:
fixed it with the locator attribute 
`    await page.locator('input[name="use-latest-snapshot"]').click();
`